src/index.js

```diff
@@ -38,11 +42,15 @@ function fetchAndLoadDataModel() {
     .then(responses => {
       if (checkLoginCredentials(responses[0], username, password, id)) {
         generateClasses(responses[0], responses[1], responses[2]);
+        displayAllTrips();
+        domUpdates.displaySidebar(user, sidebar);
         domUpdates.toggleHidden(loginPage);
         domUpdates.toggleHidden(main);
-        domUpdates.toggleHidden(navBar);
-        displayAllTrips();
-        domUpdates.displayNavBar(user, navBar);
+        domUpdates.toggleHidden(sidebar);
+        setStartMin()
+        document.querySelector('.plan-trip-button').addEventListener('click', toggleFormView)
```

Comment:
Could add event listener to page that fires functions passing the event target in to DRY up some of the code.